hnn_core/batch_simulate.py

```diff
@@ -97,9 +81,23 @@ def __init__(self, set_params, net=jones_2009_model(), tstop=170,
             If True, save calcium concentrations.
             Note, `save_outputs` must be True.
             Default: False.
+        record_vsec : 'all' | 'soma' | False
```

Comment:
While these do get checked when `simulate_dipole()` is run, I think it'd be worth adding a `check_option()` test here because you won't know that this value is incorrect until `batch_simulate.run()` is called

Reply:
Thanks for the suggestion!  `check_option()` is already in place, but I’ve added a unit test for validation to catch any issues earlier. 😉